docs/CLI.md

```diff
@@ -1,5 +1,16 @@
 # SCIP CLI Reference
 
+<!--toc:start-->
+
+- [SCIP CLI Reference](#scip-cli-reference)
+  - [`scip convert`](#scip-convert)
+  - [`scip lint`](#scip-lint)
+  - [`scip print`](#scip-print)
+  - [`scip snapshot`](#scip-snapshot)
+  - [`scip stats`](#scip-stats)
+  - [`scip json`](#scip-json)
```

Comment:
Nit: The `scip json` subcommand breaks alphabetical ordering.

Reply:
Will be removed entirely once the command becomes a flag instead